Makefile

```diff
@@ -97,6 +97,8 @@ u-boot-sunxi-with-spl.bin:
 
 cleanfast:
 	@rm -rvf build
+	@rm -rvf initramfs-pine64-pinephone/
+	@rm -rvf initramfs-pine64-pinetab/
```

Comment:
Wouldn't this do better?

`	@rm -rvf initramfs-*/`

Reply:
Yes, good idea.